e2e/test/scenarios/visualizations-tabular/visualizations-tabular-reproductions.cy.spec.js

```diff
@@ -954,6 +955,51 @@ describe.skip("issue 25415", () => {
   });
 });
 
+describe("issue 7884", () => {
+  const sourceQuestionDetails = {
+    native: {
+      // previously was "SELECT 1 AS C1, 2 AS C2, 3 AS C3"
+      query: "SELECT 1 AS C1, 3 AS C3",
+    },
+  };
```

Comment:
Did you mean to have this as a nested question query?

The test is failing because there isn't a single query with `SELECT 1 AS C1, 2 AS C2, 3 AS C3` created in here.

Reply:
Another test failed - not this one. And there was a real issue. Pushed a fix now

Reply:
I still don't see when did we create a question with `SELECT 1 AS C1, 2 AS C2, 3 AS C3` in this test?

Your screenshots clearly show that the test failed before your fix, and is now passing. But I don't understand how. What am I missing?

Reply:
I can make it more obvious with API calls. Let me do it in a sec